flycheck.el

```diff
@@ -9466,6 +9492,36 @@ This syntax checker needs Rust 1.7 or newer.  See URL
   :modes rust-mode
   :predicate flycheck-buffer-saved-p)
 
+(flycheck-define-checker rust-clippy
+  "A Rust syntax checker using clippy.
+
+See URL `https://github.com/rust-lang-nursery/rust-clippy'."
+  :command ("cargo" "\+nightly" "clippy" "--message-format=json")
```

Comment:
That `\` looks suspicious

Reply:
It does!  But the command didn't work without it.

Reply:
Huh.  That *is* strange.  AFAICT this `\` doesn't do anything: it escapes a non-special character, which is the same as not escaping it (I mean that `(string= "\+test" "+test")` evaluates to `t`).  Am I missing something?

Reply:
Hmm.  I could not reproduce the issue on this machine, so I removed the `\`.  Hopefully it was a fluke.